python/lancedb/db.py

```diff
@@ -255,6 +256,48 @@ def create_table(
             tbl = LanceTable(self, name)
         return tbl
 
+    def copy_table(
```

Comment:
Should we implement this as `Database.create_table(existed_table | dataframe | arrow_table)`? 

Feel it can reduce the surface area of the one-off request.

Reply:
actually based on the standup meeting, once the table versioning can be based on a namespace so copying will no longer be needed. Closing this